static/js/compose_actions.js

```diff
@@ -290,6 +311,13 @@ export function start(msg_type, opts) {
     // Show either stream/topic fields or "You and" field.
     show_box(msg_type, opts);
 
+    // remove any previously set click handler and reset with relevant opts
+    $("#message_type_icon").off("click");
+    $("#message_type_icon").on("click", () => {
+        toggle_message_type(opts);
+    });
```

Comment:
Another jquery question: is it reasonable to turn this off and on again each time the compose box is opened? Or should I try to find a way to set the event handler once and update `opts` in some other way?